admin-dev/filemanager/include/utils.php

```diff
@@ -64,8 +64,25 @@ function rename_folder($old_path, $name, $transliteration)
     }
 }
 
+function getFileExtension($path)
```

Comment:
Little phpDoc to help developer use it ? 😄

Reply:
I put it as internal and added some documentation.